pkg/operator/workloadcontroller/workload_controller.go

```diff
@@ -142,6 +145,21 @@ func (c OpenShiftAPIServerOperator) sync() error {
 		return nil
 	}
 
+	// block until extension-apiserver-authentication configmap is available
+	// see https://bugzilla.redhat.com/show_bug.cgi?id=1795163#c19 to check why we have to wait for it
+	// in the future we need to change upstream code to be more dynamic
```

Comment:
```
// block until extension-apiserver-authentication configmap is available to avoid 
// that openshift-apiserver starts up with request header setting (which are not dynamically reloaded).
```